### PR TITLE
Add datatables

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/datatables/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "datatables" %}
+{% set version = "1.10.16" %}
+{% set sha256 = "3f4e44350bb80b6f4afa5d2841ff14fc6aabe176b6e51e3c3fe0324b040ebf73" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataTables/DataTables/archive/1.10.16.tar.gz
```
Comment (dougalsutherland) [inline recipes/datatables/meta.yaml]: `/archive/{{ version }}.tar.gz`

File: recipes/datatables/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
```
Comment (dougalsutherland) [inline recipes/datatables/meta.yaml]: `noarch: generic` please.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge.

File: recipes/datatables/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
```
Comment (isuruf) [inline recipes/datatables/meta.yaml]: Can you please fix this as in the rest of the PRs?

Comment (sannykr) [reply recipes/datatables/meta.yaml]: done

Comment (sannykr) [reply recipes/datatables/meta.yaml]: @isuruf : It has passed all checks. Please let me know if you have any more suggestions.

**Pull request merged**